### ISSUE TITLE
Pie menu sliding away during editing

### ISSUE THREAD

**Issue by tcql** [wojtryb/Shortcut-Composer#51]
**General info**
- Plugin version: 1.3.2 (and development)
- Operating System: Windows 10

**Bug description**
When working on my tablet display, if I try to edit pie menu items, the pie... uhh... runs away? when an item is hovered over it. 

I'm attaching a quick video:

https://github.com/wojtryb/Shortcut-Composer/assets/348805/2d4437c3-241d-4a82-b4d2-239f08af2276

Other relevant info: my tablet is a Huion Kamvas Pro 13. I originally had it set up as a separate display, when this behavior was observed. If I clone my primary monitor so the tablet and monitor are mirroring eachother, the behavior no longer occurs.

**Comment by wojtryb** [comment]
Hi, thank you for reporting!

The bug is very likely related to the multi display setup. I admit that the plugin was never tested with more than one monitor.

I'll try to recreate this setup, but can't really guarantee I'll be able to trace it and fix it.

Things I'd like to know:
- do your two displays have a different resolution? If so, does it still occur when the bigger screen resolution is changed, so they are the same?
- does the bug occur on krita 5.2 which uses newer version of tools that interact with windows10 desktop? (Can be [downloaded here](https://binary-factory.kde.org/job/Krita_Nightly_Windows_Build/lastSuccessfulBuild/artifact/krita-nightly-x64-5.2.0-prealpha-d9959dfda2-dbg.zip) - it's a direct link for portable version)

Also - I see another multi display related bug here - the labels in ScrollArea are too small - as if the settings were displayed in the other screen (which I assume - has a different resolution)

**Comment by tcql** [comment]
@wojtryb Sorry for the delay responding. The issue does still occur in 5.2, _but_ I found the cause!

All my monitors / tablet display are the same resolution (1920x1080) but since the tablet is a smaller physical size, Windows automatically applied scaling so text would be less tiny
 
![image](https://github.com/wojtryb/Shortcut-Composer/assets/348805/14938655-4e00-448a-a2e7-c9c1b793b2c1)

If I set scaling to 100%, the pie menus don't slide away, however the app text and UI elements are quite small so it would be nice if there's a way for the plugin to work still even with scaling applied. Even if there isn't a reasonable fix for this, thanks for the plugin and for looking at this 🙇
